README.md

```diff
@@ -1,3 +1,7 @@
+# FlowMagic-SDK
+
+A description of this package.
```

Comment:
"description of this package" sounds like a placeholder.  You really want to add that?

Reply:
Removed.